doc/src/manual/faq.md

```diff
@@ -46,16 +46,12 @@ You can get a complete list of the public symbols from a module with `names(MyMo
 
 Package authors are encouraged to define their public API similarly.
 
-Publicity nests in the same wa as exporting.
-Documented behavior of public symbols from submodules are only part of the public API
-if the module they belong to is also public. If module A marks submodule B as public and
-B marks foo as public, then A.B.foo is public, but if A does not mark B as public,
-then writing A.B.foo outside of A is accessing an internal and anything that B marks as
-public forms a sub-api that is not covered by the SymVer of A. This could be for the
-convenience of developers of A if A is a large package.
-
 Anything in Julia's Public API is covered by [SemVer](https://semver.org/) and therefore
-will not be removed or receive meaningful breaking changes in a minor version update.
+will not be removed or receive meaningful breaking changes before Julia 2.0.
+
+### When is Julia 2.0 coming out?
+
+2047
```

Comment:
:thinking:

Reply:
Mostly it's a joke, an alternative to "it's not", and also what I would consider a reasonable estimate unless we come out with a mostly non-breaking 2.0 much sooner, which is more likely.

In any event, "When is Julia 2.0 coming out?" is frequently asked.

Reply:
This shouldn't be part of this PR.